numpy/lib/npyio.py

```diff
@@ -506,6 +506,21 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
     Notes
     -----
     For a description of the ``.npy`` format, see :py:mod:`numpy.lib.format`.
+        
+    Any data saved to the file is automatically added to the end if the file is still opened. 
+    It has similar behavior to append not overwrite since the file is not closed yet.
```

Comment:
Perhaps more concisely "Any data saved to the file is appended to the end of the file". If the file is closed, clearly writing will fail, and the second sentence does not contribute much information.

Reply:
@mattip  Thanks for your feedback, I have updated it.